go.mod

```diff
@@ -3,9 +3,12 @@ module github.com/pion/ion-load-tool
 go 1.13
 
 require (
-	github.com/cloudwebrtc/go-protoo v0.0.0-20200510140124-cf4744e87257
+	github.com/cloudwebrtc/go-protoo v0.0.0-20200602160428-0a199e23f7e0
 	github.com/google/uuid v1.1.2
 	github.com/pion/ion v0.3.5
 	github.com/pion/producer v0.0.0-20200517154709-fabafe00b1f3
 	github.com/pion/webrtc/v2 v2.2.24
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 )
+
```

Comment:
@billylindeman I don't fully understand how module versioning work with go, so I currently use local ion checkout. Can we somehow point pion/ion dependency to our repo master-tandem with your authentication changes?

Reply:
Not really without changing the dependency entirely